gdx/src/com/badlogic/gdx/graphics/g2d/GlyphLayout.java

```diff
@@ -45,7 +45,7 @@
  * @author Alexander Dorokhov
  * @author Thomas Creutzenberg */
 public class GlyphLayout implements Poolable {
-	static private final Pool<GlyphRun> glyphRunPool = Pools.get(GlyphRun::new);
+	static private final Pool<GlyphRun> glyphRunPool = Pools.get(GlyphRun.class);
```

Comment:
Maybe this is intended for a follow-up pr, but places like this could be easily replaced with a `new DefaultPool(GlyphRun::new)`, right?

Reply:
Yes, as mentioned in the other comment thread on this PR it would be better to do separately.

https://github.com/libgdx/libgdx/pull/7648#discussion_r2104114250